ExternalTasks/NodeJs/main.js

```diff
@@ -13,7 +13,7 @@ const identity = {
     token: 'ZHVtbXlfdG9rZW4=',
 };
 
-const TOPIC = 'Etikett-ausdrucken';
+const TOPIC = 'AktivierungsemailSenden';
 const MAX_TASKS = 10;
 const POLLING_TIMEOUT = 1000;
 const WAIT_TIMEOUT = 10000;
```

Comment:
```suggestion
const externalTaskProcessingTimeout = 10000;
```

Find ich passender, weil man dann drekt versteht wofür diese Konstante gut ist.